cfme/tests/cli/test_appliance_console.py

```diff
@@ -1601,3 +1601,30 @@ def test_appliance_console_vmdb_log_negative(temp_appliance_preconfig_funcscope)
     assert "Validated failed with error" in result[-1], "Database Configuration is not failed"
 
     ap_console_tail.validate(wait="60s")
+
+
+@pytest.mark.manual
```

Comment:
Add pytest meta markers for BZ.

Reply:
Add tier and `test_requirements` if not already set by the global pytest marker.

Reply:
**test_requirements** is already there now set tier()